src/engine/controls/loopingcontrol.h

```diff
@@ -73,8 +73,8 @@ class LoopingControl : public EngineControl {
     };
 
     struct LoopInfo {
-        mixxx::audio::FramePos startPosition = mixxx::audio::kInvalidFramePos;
-        mixxx::audio::FramePos endPosition = mixxx::audio::kInvalidFramePos;
+        mixxx::audio::FramePos startPosition;
+        mixxx::audio::FramePos endPosition;
```

Comment:
What's the benefit of this change? The previous version made the default value explicit.